assets/scss/components/stencil/heroCarousel/_heroCarousel.scss

```diff
@@ -84,6 +83,13 @@
             @include carouselOpaqueBackgrounds($slick-dot-bgColor);
         }
     }
+
+    @media (max-width: 550px) and (max-height: 550px) {
+        .heroCarousel-content {
+            display: none;
+        }
+    }
```

Comment:
Why do we need to hide content here?

Reply:
As you can see media values are quite specific - on this aspect ratio we still have width of mobile version but no enough place for content because of small height. So I decided to remove text for such cases. It can affect devices like iPhone 4s